src/controller/task.ts

```diff
@@ -32,6 +37,11 @@ export default {
           .limit(limit)
           .lean()
       ])
+      Logger.info('Total number of tasks are', {
```

Comment:
Usually logs should be written on write operations POST/PUT/PATCH/DELETE but not GET. 
It's redundant to write logs on get operations.

Reply:
fixed e9acee3ca21fc4f0db8449487a6402632f34093e